### DISCUSSION TITLE
How to change the hardcoded public servers and pub key

### DISCUSSION THREAD

**Discussion by smartekIT** [rustdesk/rustdesk#1001] [Q&A]
First thanks for the great project,
I can see the public servers are hard coded in <pre>hbb_common/././config.rs</pre>
and the pub key also hard coded in <pre>src/client.rs</pre>

now if I change the above I get the popular error: Key Mismatch.
in my self-hosted rustdesk-server I use the option: <pre>-k _</pre>

how can I keep the <pre>-k _</pre> and not getting the error message on the client <pre>Key Mismatch</pre> when I change the hardcoded key and public servers mentioned above ?

Thanks heaps

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/blob/1a4f0507a2f19d1319442ca81c7220c9fcedcfa7/libs/hbb_common/src/config.rs#L68

**Comment by smartekIT** [comment]
Thanks champion, but when I change the 3 servers listed there, and the server key as well. I get the error key Mismatch on the client side if the hosted server is using -k _ 
to get around the error i had to remove -k _ from the server side so client doesn't generate that error !

the question is: when i change the 3 servers and the public key in config.rs , do i have to adjust any other settings like encryption settings anywhere? and can i have 5 server there instead of 3 and not getting errors?

Thanks again.
